ui/raidboss/data/triggers/ultima_weapon_ultimate.js

```diff
@@ -8,6 +8,7 @@
     {
       id: 'UWU Feather Rain',
       regex: /Feather Rain/,
+      regexDe: /Federregen/,
```

Comment:
Just FYI, timeline triggers don't need to be translated.  I'll edit this myself and then merge.